src/Authenticators/LDAPAuthenticator.php

```diff
@@ -163,18 +166,39 @@ public static function authenticate($data, Form $form = null)
      * Try to authenticate using the fallback authenticator.
      *
      * @param array $data
-     * @param null|Form $form
+     * @param HTTPRequest $request
      * @return null|Member
+     * @internal param null|Form $form
```

Comment:
I'm not sure what the purpose of this is?

Reply:
Yeah, that is weird. And I see it was introduced by me. Must've been an accidental paste. Will remove.